src/Test.FunctionalTests.Sarif/TestData/Multitool/ValidateCommand/Inputs/SARIF1019.RuleIdMustBePresentAndConsistent_Valid.sarif

```diff
@@ -0,0 +1,79 @@
+{
+  "$schema": "https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.5.json",
+  "version": "2.1.0",
+  "runs": [
+    {
+      "tool": {
+        "driver": {
+          "name": "Sarif Functional Testing"
+        }
+      },
+      "logicalLocations": [
+        {
+          "name": "A"
+        },
+        {
+          "name": "B"
+        }
+      ],
+      "results": [
+        {
+          "ruleId": "TST0001",
+          "message": {
+            "text": "Message."
```

Comment:
>Message [](start = 21, length = 7)

You can use this property to make each test case self-describing. For example: "This result contains only a ruleId property.", "This result contains only a rule.id property", "This result contains both a ruleId and a rule.id property, and they are the same".

Similarly, in the Invalid file, you can describe your result: "This result contains both a ruleId and a rule.id property, but they are different."